ginkgo_me/BBQ_gotest_test.go

```diff
@@ -0,0 +1,152 @@
+package ginkgo_me
```

Comment:
plz add your examples with Readme explanation to different dir (with appropriative maming to make it's clear what this example shows)